docs/docs/test-runner/browser-launchers/browserstack.md

```diff
@@ -81,3 +91,5 @@ The Browserstack launcher takes two properties, `capabilities` and `localOptions
 `capabilities` are the selenium capabilities used to configure the browser to launch in Browserstack. You can generate most of these on the Saucelabs. It must contain a `browserstack.user` and `browserstack.key` property to authenticate with Browserstack, as well as `name`, `build` and `project` to identify the test run.
```

Comment:
See the line above the lists properties.

Reply:
Fixed